packages/flutter/test/material/tabs_test.dart

```diff
@@ -386,6 +386,54 @@ void main() {
       material3 ? const Size(14.25, 72.0) : const Size(14.0, 72.0));
   });
 
+  testWidgets('Material2 - Tab icon-label margin', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData(useMaterial3: false);
+    Widget buildApp() {
+      return MaterialApp(
+        theme: theme,
+        home: const Center(child: Material(
+          child: Tab(
+            icon: Icon(Icons.house),
+            text: 'x',
+          ),
+        ),
+      ));
+    }
+
+    double getIconMargin() {
+      final Rect iconRect = tester.getRect(find.byIcon(Icons.house));
+      final Rect labelRect = tester.getRect(find.text('x'));
+      return labelRect.top - iconRect.bottom;
+    }
+
+    await tester.pumpWidget(buildApp());
+    expect(getIconMargin(), 10);
+  });
+
+  testWidgets('Material3 - Tab icon-label margin', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData();
+    Widget buildApp() {
+      return MaterialApp(
+        theme: theme,
+        home: const Center(child: Material(
+          child: Tab(
+            icon: Icon(Icons.house),
+            text: 'x',
+          ),
+        ),
+      ));
+    }
```

Comment:
```suggestion
    await tester.pumpWidget(const MaterialApp(
      home: Material(
        child: Tab(
          icon: Icon(Icons.house),
          text: 'x',
        ),
      ),
    ));
```

We don't need variable if we're not testing any theme property and since M3 is default, we don't even need to specify theme here.